docs/source/blogs/glowgr-blog/glowgr-blog.rst

```diff
@@ -0,0 +1,226 @@
+========================================================================================================
+Introducing GloWGR: An industrial-scale, ultra-fast and sensitive method for genetic association studies
+========================================================================================================
+
+Authors:
+`Leland Barnard <https://github.com/LelandBarnard>`_,
+`Henry Davidge <https://github.com/henrydavidge>`_,
+`Karen Feng <https://github.com/karenfeng>`_,
+`Joelle Mbatchou <https://github.com/joellesophya>`_,
+`Boris Boutkov <https://github.com/bboutkov>`_,
+`Kiavash Kianfar <https://github.com/kianfar77>`_,
+`Lukas Habegger <https://github.com/habeggel>`_,
+`Jonathan Marchini <https://github.com/jmarchini>`_,
+`Jeffrey Reid <https://github.com/jgreid>`_,
+`Evan Maxwell <https://github.com/emaxwell>`_,
+`Frank Austin Nothaft <https://github.com/fnothaft>`_
+
+June 22, 2020
+
+*The industry partnership between Regeneron and Databricks is
+enabling innovations in genomics data processing and analysis. Today, we
+announce that we are making this work available to the open source
```

Comment:
Mistake on my end, can you change `this work` to `a new whole genome regression method`?